tests/ctst/steps/azureArchive.ts

```diff
@@ -330,30 +330,63 @@ Then('manifest containing object {string} should contain {int} objects',
         assert.strictEqual(count, objectCount);
     });
 
-Then('blob for object {string} must be rehydrated', async function (this: Zenko, objectName: string) {
-    let found = false;
-    const {
-        tarName,
-    } = await findObjectPackAndManifest(
-        this,
-        this.getSaved<string>('bucketName'),
-        objectName || this.getSaved<string>('objectName'),
-    );
-    assert(tarName);
-    while (!found) {
-        found = await AzureHelper.blobExists(
-            this.parameters.azureArchiveContainer,
-            `rehydrate/${tarName}`,
-            getAzureCreds(this),
+Then('blob for object {string} must be rehydrated {string} notifying the queue',
+    async function (this: Zenko, objectName: string, notifyQueue: string) {
+        let found = false;
+        const {
+            tarName,
+        } = await findObjectPackAndManifest(
+            this,
+            this.getSaved<string>('bucketName'),
+            objectName || this.getSaved<string>('objectName'),
         );
-    }
-    await AzureHelper.sendBlobCreatedEventToQueue(
-        this.parameters.azureArchiveQueue,
-        this.parameters.azureArchiveContainer,
-        `rehydrate/${tarName}`,
-        getAzureCreds(this),
-    );
-});
+        assert(tarName);
+        while (!found) {
```

Comment:
shouldn't we have a timeout here?
or is it handled by CTST already?

Reply:
Yes it is already handled by CTST